### PR TITLE
Add DSH_URL and HUB_URL to CMS creation

### PR DESCRIPTION
**Pull request by thescubageek**
* Adds `DSH_URL` and `HUB_URL` to CMS creation
* Removes deprecated `WIDGET_GARDEN_URL`, `THEME_GARDEN_URL`, and `LAYOUT_GARDEN_URL` env vars

### CODE REVIEW

File: config/defaults.yml [hunk 1]
```diff
@@ -26,6 +26,8 @@ deploys:
         - "BITBUCKET_USER"
         - "PROVISIONER_URL"
         - "REDIS_PROVIDER"
+        - "DSH_URL"
```
Comment (variousred) [inline config/defaults.yml]: do you need cxm_hostname in here as well?

Comment (thescubageek) [inline config/defaults.yml]: @variousred I don't think so, it's not required by [G5 Header](https://github.com/g5search/g5_header#upgrade-guide) -- unless it's used by some other service I'm unaware of...

Comment (variousred) [inline config/defaults.yml]: do you need it in the dynamic method definition list then?

Comment (thescubageek) [inline config/defaults.yml]: What is `CXM_HOSTNAME` used for?

Comment (variousred) [inline config/defaults.yml]: No idea. I saw it in green in `client_app` — only just now realizing its in red as well; hence you didn't add it.

Comment (thescubageek) [inline config/defaults.yml]: Oh, right, it's the trailing comma that makes it look like an add.

**Comment by variousred** [comment]
:shipit:

**Pull request merged**